spec/models/stripe_wrapper_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'spec_helper'
+
+describe StripeWrapper::Charge do
+  describe ".create", :vcr do
+    let(:token) { generate_stripe_token }
+
+    before { StripeWrapper.set_api_key }
+
+    subject do
```

Comment:
you can do `subject(:charge) ...` here and instead of `subject` in examples you can call `charge` to reflect what subject is.